lib/views/public_body/_more_info.html.erb

```diff
@@ -19,7 +19,7 @@
                     "https://www.oscr.org.uk/about-charities/search-the-register/charity-details?number=#{ tag_value }" %><br>
     <% else %>
       <%= link_to _('Charity registration'),
-                    "http://beta.charitycommission.gov.uk/charity-details/?regid=#{ tag_value }&subid=0" %><br>
+                    "https://register-of-charities.charitycommission.gov.uk/charity-search/-/results/page/1/delta/20/keywords/#{ tag_value }" %><br>
```

Comment:
OOI, why not link [directly to the page for the charity](https://register-of-charities.charitycommission.gov.uk/charity-search/-/charity-details/507221), rather than the [search page](https://register-of-charities.charitycommission.gov.uk/charity-search/-/results/page/1/delta/20/keywords/507221)?

Reply:
Doh, sorry, missed the [comment above](https://github.com/mysociety/whatdotheyknow-theme/pull/732#issuecomment-728146916)

Reply:
It's a reasonable question… unfortunately I've not had my coffee yet, so here's some unstructured thoughts:

If we link directly to the page for the charity itself, there is a chance we're not actually getting the right body.

In the example at https://github.com/mysociety/whatdotheyknow-theme/issues/711#issuecomment-728145412 there are multiple bodies that use the same charity number (which then has a sequential identifier applied to it). If we pass the charity number to the _direct_ page the system will not render any of these 'linked' bodies. This appears to be important, because it could be a 'linked' body that we list, rather than the parent.

There is a way to locate the body within the page displayed, but it is messy, since it requires multiple steps to be followed (click on "Linked Charities", then click the name of the charity, then click "Linked charity $number"). If we render the search page instead, the user gets a choice as to which one they want.

I'm agnostic on this - the solution evolved slightly since the patch was initially written; if there's an easier way that we think will be sufficient I'd be happy to go with that instead. 👍